_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,173 @@
+    extended the previous half-signed proposal and then provided his own
+    proposal that was equivalently effective but simpler.  The new
+    proposal has the initiator create and sign (but not broadcast) a
+    transaction that spends their UTXO back to themselves.  They give
+    this to the other party as a proof of good faith---if the initiator
+    later fails to sign the actual funding transaction, the respondent
+    can broadcast the good-faith transaction, forcing the initiator to
+    pay an onchain fee.  Fournier concludes his post with a summary of
+    the tradeoffs between the different approaches.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME:jonatack
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="http://FIXME"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [Bitcoin Core 0.21.0rc5][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.
+
+- [LND 0.12.0-beta.rc5][LND 0.12.0-beta] is the latest release candidate
+  for the next major version of this LN node.  It makes [anchor
+  outputs][topic anchor outputs] the default for commitment transactions
+  and adds support for them in its [watchtower][topic watchtowers]
+  implementation, reducing costs and increasing safety.  The release
+  also adds generic support for creating and signing [PSBTs][topic psbt]
+  and includes several bug fixes.
+
+{% comment %}<!--
+- Bitcoin Core 0.20.2rc1 and 0.19.2rc1 are expected to be
+  [available][bitcoincore.org/bin] sometime after the publication of
+  this newsletter.  They contain several bug fixes, such as an
+  improvement described in [Newsletter #110][news110 bcc19620] that will
+  prevent them from redownloading future taproot transactions that they
+  don't understand.
+-->{% endcomment %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18077][] introduces support for a common automatic port
+  forwarding protocol, [NAT-PMP][rfc 6886] (Network Address Translation Port
+  Mapping Protocol).  A listening Bitcoin client started with the `-natpmp`
```

Comment:
> `-natpmp`

What is the appropriate formatting or phrasing to allude to start-up options?

Reply:
For `-foo`, I usually call it a "configuration parameter".  However, for `-foo=bar`, I call `-foo` a "configuration option" and `bar` a parameter.  Although that makes sense inside my head, I'm not sure it's consistent to readers and so maybe this is something we should figure out for the style guide.

Reply:
I've been following that terminology in my patches since you explained those to me some time age.